### DISCUSSION TITLE
"npm start" not working, how to fix?

### DISCUSSION THREAD

**Discussion by johnliu-code** [krjo/shopify-webpack-dev-workflow#19] [Q&A]
Hi,
Thank you so much for this workflow.
I have tried to set-up my Windows 10 with this, but it doesn't work. Get some errors.
I did a lot of searches on google, but could not fix it. 
Please help me if there is some way can fix this, appreciate!

1. npm install, get this：
npm ERR! code 1
npm ERR! path C:\www\Shopify\shopify-workflow\node_modules\node-sass
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node-gyp rebuild
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@3.8.0
npm ERR! gyp info using node@15.12.0 | win32 | x64
npm ERR! gyp ERR! configure error
npm ERR! gyp ERR! stack Error: Command failed: C:\Python39\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
npm ERR! gyp ERR! stack   File "<string>", line 1
npm ERR! gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
npm ERR! gyp ERR! stack                       ^
npm ERR! gyp ERR! stack SyntaxError: invalid syntax
npm ERR! gyp ERR! stack 
npm ERR! gyp ERR! stack     at ChildProcess.exithandler (node:child_process:326:12)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:369:20)
npm ERR! gyp ERR! stack     at maybeClose (node:internal/child_process:1067:16)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
npm ERR! gyp ERR! System Windows_NT 10.0.19041
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\www\\Shopify\\shopify-workflow\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd C:\www\Shopify\shopify-workflow\node_modules\node-sass
npm ERR! gyp ERR! node -v v15.12.0
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jeanl\AppData\Local\npm-cache\_logs\2021-03-18T18_41_41_664Z-debug.log 

2. npm start, get this:

> shopify-webpack-theme-development@1.0.0 start
> NODE_ENV=development webpack --watch

'webpack' is not recognized as an internal or external command,
operable program or batch file.
Note: This command was run via npm module 'win-node-env'
>> node foo.js
node:internal/modules/cjs/loader:927
  throw err;
  ^

Error: Cannot find module 'C:\www\Shopify\shopify-workflow\foo.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)      
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

**Comment by krjo** [comment]
Check what node version you're using with node -v. I'm currently using the latest release 14.16.0 successfully.
